test/visitor/json.cpp

```diff
@@ -27,6 +28,8 @@ using nmodl::parser::NmodlDriver;
 std::string run_json_visitor(const std::string& text, bool compact = false) {
     NmodlDriver driver;
     auto ast = driver.parse_string(text);
+
+
```

Comment:
nitpick : extra line

Reply:
I ran

`make clang-format cmake-format`

why wasn't it corrected?